### ISSUE TITLE
eval.log output

### ISSUE THREAD

**Issue by AlexMRuch** [awslabs/dgl-ke#92]
What's the output of `eval.log` supposed to be? When I run this on my user-defined knowledge graph, it just saves an empty text file.

Here's an example of the script I run for eval:
```
DGLBACKEND=pytorch dglke_eval \
--data_path results_SXSW_2018 --dataset SXSW2018 \
--data_files entities.tsv relations.tsv all_ctups_40.tsv valid.tsv test.tsv --format udd_hrt \
--model_name ComplEx \
--hidden_dim 128 --gamma 128 \
--mix_cpu_gpu --num_proc 6 --num_thread 5 --gpu 0 1 \
--batch_size_eval 1024 --neg_sample_size_eval 10000 --eval_percent 20 \
--model_path /home/amruch/graphika/kg/ckpts/SXSW2018_ComplEx_20200511/
```

**Comment by classicsong** [comment]
The logger should be deprecated. You can Ignore it.

**Comment by classicsong** [comment]
Fixed in PR #104
